src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Box, Button, GU, TextInput, Text, Info, IconMail } from '@aragon/ui'
+import { login } from './notification-service-api'
+import { AUTH_PREVERIFY } from './constants'
+import { getEthNetworkType } from '../../../local-settings'
+import notificationSvg from './notifications.svg'
+import checkmarkSvg from './check-mark.svg'
+
+export default function NotificationsLogin({
+  dao,
+  authState,
+  email,
+  onEmailChange,
+}) {
+  const [inputEmail, setInputEmail] = useState('')
+  const [apiError, setApiError] = useState(null)
+  const network = getEthNetworkType()
+
+  const handleEmailChange = e => {
+    setInputEmail(e.target.value)
+  }
+
+  const handleLogin = async e => {
+    try {
+      await login({ email: inputEmail, dao, network })
+      onEmailChange(inputEmail)
+    } catch (e) {
+      setApiError(e.message)
+      console.error('Failed to login', e)
+    }
+  }
+
+  if (authState === AUTH_PREVERIFY) {
+    return (
+      <Box heading="Email notifications">
+        <NotificationImage />
+        <div>
+          <Checkmark />
+          <Text>Awaiting verification. Please check your email!</Text>
```

Comment:
This needs some alignment and positioning:
![image](https://user-images.githubusercontent.com/3072458/62777399-b3ca5100-baad-11e9-8dae-fa083503275f.png)

Reply:
We are using this in several places, we may need to consider a component for it:
![image](https://user-images.githubusercontent.com/3072458/62777479-f9871980-baad-11e9-8470-fe65e722ebb8.png)
cc @bpierre 
[This](https://github.com/aragon/aragon-apps/pull/935/files#diff-eb9825de0d64a86baaf26b18f7dfe25bR59) is the piece of code for `Voting`, it may come in handy @2color